backend/routes/expectedWaitTime.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const admin = require('firebase-admin');
+
+// Expected wait time endpoint
+router.post('/expectedWaitTime', async (req, res) => {
+    try {
+        // Get the location from the request body.
+        const location = req.body.location
+
```

Comment:
Add error check if location is not present. Return error code 400 "location required"